modules/odc-visualizing-the-binding-connections-to-resources.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * applications/connecting_applications_to_services/odc-connecting-an-application-to-a-service-using-the-developer-perspective.adoc
+:_content-type: PROCEDURE
+[id="odc-visualizing-the-binding-connections-to-resources_{context}"]
+= Visualizing the binding connections to resources
+
+As a user, you can visualize a Service Binding through *Label Selector* in *Topology* view. Label selector is a feature added to Service Binding Operator that aims to simplify the process of binding applications to backing services. It does this by allowing developers to use specified labels to find and connect applications instead of using the name of the application. It aims to alleviate the confusing nature of one or many services to many application binds. This is done by specifying labels that will be consumed by the Service Binding Operator when creating `ServiceBinding` resources.
```

Comment:
```suggestion
As a user, you can visualize a Service Binding through *Label Selector* in *Topology* view. Label selector is a feature added to Service Binding Operator that aims to simplify the process of binding applications to backing services. It does this by allowing developers to use specified labels to find and connect applications instead of using the name of the application. It aims to alleviate the confusing nature of one or many services to many application binds. This is done by specifying labels when when creating `ServiceBinding` resources that will be consumed by the Service Binding Operator to find the applications to create a service binding with.
```